ILL/IDF/d7_generateIDF.py

```diff
@@ -0,0 +1,120 @@
+import os
+path = os.path.abspath("")
+import sys
+sys.path.insert(0, path)
+from helper import MantidGeom
+
+from lxml import etree as le
+# using metre as unit
+instrumentName = 'D7'
+validFrom = "1900-01-31 23:59:59"
+validTo="2100-01-31 23:59:59"
+
+# Monochromator position
+source = -3.6
+
+# Fermi chopper
+zChopper = -0.48
+
+# 2 Monitors
+zMon1 = -1.3
+zMon2 = 1.5
+
+# distance between the sample and detectors:
+rSampleDetectors = 1.5177 # distance from the sample of even detectors
+minTwoTheta = 0
+maxTwoTheta = 136
+
+# definition of a detector bank
+numberBanks = 3 # banks 2, 3 and 4; bank 1 is a monitor
+numberDetectors = 44
+
+# definition of a pixel
+tubeRadius = 0.0125
```

Comment:
I think he said it's 25.4 mm, so `0.01252` ?

Reply:
That would be the precise value, will update.